.github/ISSUE_TEMPLATE/request-for-change.md

```diff
@@ -0,0 +1,13 @@
+---
+name: DCR Request for change
+about: Tell us about a feature that you're working on
+title: "[RFC]"
+labels: RFC, dotcom-rendering
+assignees: ''
+
+---
+
+The Dotcom team would like to help you with your proposed change. Please provide us with a brief and concise explanation for what you're looking to do we will try to make the process easier with early feedback.
+
+**Who is responsible for completing this RFC process?**
```

Comment:
Will this change be enforced by repository rules / codeowners configuration, or will we rely on Dotcom team members commenting on PRs and nudging them towards the RFC process?

Reply:
Initially, we should keep the flow optional and only start to enforce things as we learn more about usage